bears/coffee_script/CoffeeLintBear.py

```diff
@@ -22,11 +23,280 @@ class CoffeeLintBear:
     CAN_DETECT = {'Syntax', 'Formatting', 'Smell'}
 
     severity_map = {'warn': RESULT_SEVERITY.NORMAL,
-                    'error': RESULT_SEVERITY.MAJOR}
+                    'error': RESULT_SEVERITY.MAJOR,
+                    'ignore': RESULT_SEVERITY.INFO}
 
     @staticmethod
     def create_arguments(filename, file, config_file):
-        return '--reporter=raw', filename
+        return '--reporter=raw', '-f', config_file, filename
+
+    @staticmethod
+    def generate_config(filename, file,
+                        max_line_length: int=79,
+                        max_line_length_affect_comments: bool=True,
+                        space_before_and_after_arrow: bool=True,
```

Comment:
i think having ``a <- 1`` is better than having ``a<-1``

2016-07-14 16:10 UTC+01:00, Tushar <notifications@github.com>:
>>
>>      @staticmethod
>>      def create_arguments(filename, file, config_file):
>> -        return '--reporter=raw', filename
>> +        return '--reporter=raw', '-f', config_file, filename
>> +
>> +    @staticmethod
>> +    def generate_config(filename, file,
>> +                        max_line_length: int=79,
>> +                        max_line_length_affect_comments: bool=True,
>> +                        space_before_and_after_arrow: bool=True,
>
> I think as per the linter, the default here is to `ignore` or `False` .
> @Asnelchristian You sure to keep it as `error` and not `False`?
>
> ---
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/coala-analyzer/coala-bears/pull/555/files/f0b436c93032c451453f0a650f4c8c342e427fa3#r70823429